<launch>

  <!-- Launch Corvis Filter Node  -->
  <!-- launch-prefix="xterm -e gdb \-\-args" for debugging -->

  <!-- launch-prefix="xterm -e gdb \-\-args" for debugging -->
  <!-- launch-prefix="xterm -ex gdb \-\-args" for debugging while executing on start -->
  <!-- launch-prefix="valgrind" for using valgrind -->
  <!-- launch-prefix="GLOG_v=0 GLOG_logtostderr=-1 GLOG_log_dir=log" logging options -->

  <node pkg="xivo" type="mynode" name="xivo" output="screen" >
    <param name="config_path" value="$(env XIVO_ROOT)/cfg/tumvi_cam0.json"/>
    <param name="viewer_config_path" value="$(env XIVO_ROOT)/cfg/viewer.json"/>
    <param name="estimator_queue_size" value="1000" />
    <param name="image_topic"   value="/cam0/image_raw" />
    <param name="imu_topic"   value="/imu0" />
    <param name="viewer_type" value="pangolin" />
    <!-- <param name="viewer_type" value="ros" /> -->
    <param name="publish_state" value="true" type="bool" />
    <param name="publish_full_state" value="true" type="bool" />
    <param name="publish_map" value="true" type="bool" />
    <param name="publish_2dnav_state" value="true" type="bool" />
  </node>

  <!--
  <node pkg="image_view" type="image_view" name="image_view" launch-prefix="nice -n -20" >
    <remap from="image" to="TrackerView" />
  </node>
  -->


</launch>
